eng/pipelines/templates/jobs/ci.yml

```diff
@@ -139,7 +160,24 @@ jobs:
       vmImage: MMSUbuntu18.04
 
     steps:
+      - task: UsePythonVersion@0
+        displayName: 'Use Python 3.6'
+        inputs:
+          versionSpec: '3.6'
+
+      - template: /eng/common/pipelines/templates/steps/sparse-checkout.yml
+        parameters:
+          Paths:
+            - 'sdk/${{ parameters.ServiceDirectory }}'
+            - '**/*.xml'
+            - '**/*.md'
+            - '.vscode/cspell.json'
+            - '!sdk/**/test-recordings'
+            - '!sdk/**/session-records'
+
       - template: /eng/common/pipelines/templates/steps/check-spelling.yml
+        parameters:
```

Comment:
What this not working before?

Reply:
It works fine without specifying this parameter (as it matches the default), but since the cspell path is defined above in the sparse checkout includes, I thought it would help prevent scenarios where someone may delete/change one without the other, or wonder where the sparse checkout cspell.json include is being used.

Reply:
might be worth a comment otherwise someone will likely remove it if they see that it matches the default value.